keybot/main.go

```diff
@@ -40,6 +43,10 @@ func kingpinHandler(args []string) (string, error) {
 
 	cancel := build.Command("cancel", "Cancel any existing builds")
 
+	buildWindows := build.Command("windows", "Start a windows build")
+	buildWindowsTest := build.Command("windowsTest", "Start a windows test build")
+	cancelWindows := build.Command("cancelWindows", "Cancel last windows build")
```

Comment:
Ah ok.. yeah you can leave for now... I think we're gonna change to:

!keybot build darwin
!keybot build darwin cancel
!keybot build darwin test

We can refactor this later though